### PR TITLE
 Fix bug related to the --ampls-resource-id option in the `az aks enable-addons` command

### PR DESCRIPTION
**Pull request by ganga1980**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [x] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Review by FumingZhang — COMMENTED** [review]
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=100005829&view=results) to validate the change, test passed!

- test_aks_create_with_monitoring_aad_auth_with_highlogscale
- test_aks_create_with_private_cluster_with_monitoring_aad_auth_msi_with_ampls
- test_aks_create_with_private_cluster_with_monitoring_aad_auth_msi_with_ampls_with_highlogscale

File: src/aks-preview/HISTORY.rst [hunk 1]
```diff
@@ -13,6 +13,10 @@ Pending
 +++++++
 * Vendor new SDK and bump API version to 2024-06-02-preview.
```
[the remark this answers is not in the document]
Comment (ganga1980) [reply src/aks-preview/HISTORY.rst:14]: done.

**Review by FumingZhang — APPROVED** [review]

**Comment by ganga1980** [comment]
Hi, @yanzhudd , can you please help on getting this PR merged?

File: src/aks-preview/HISTORY.rst [hunk 2]
```diff
@@ -13,6 +13,11 @@ Pending
...
 
+7.0.0b4
+++++++++
+* Fix bug related to the -ampls-resource-id option in the `az aks enable-addons` command.
```
Comment (yanzhudd) [inline src/aks-preview/HISTORY.rst:18]:
```suggestion
* Fix bug related to the --ampls-resource-id option in the `az aks enable-addons` command.
```

**Review by yanzhudd — APPROVED** [review]

**Pull request merged**